### ISSUE TITLE
Suggestion - more than 9 items with scroll

### ISSUE THREAD

**Issue by noam2** [Nightonke/BoomMenu#125]
Hey
I have an idea for making this library even better - maybe you could make it possible to boom more than 9 items ,and have the window scroll if they are more than 9

**Comment by Nightonke** [comment]
Yep, that's a nice feature about the scrolling window which will be supported in next version. And in fact, you can have boom-buttons as many as you want and customize their positions since [version 2.1.0](https://github.com/Nightonke/BoomMenu/wiki/Version-History#v210). Check [Custom Position Wiki](https://github.com/Nightonke/BoomMenu/wiki/Custom-Position) for more information.

**Comment by noam2** [comment]
Great man ! thanks for this awesome library .. when do you expect the next version to be released?

**Comment by noam2** [comment]
another thing - version 2.1.0 is very laggy , i updated it and my toolbar ham button doesn't boom at smooth as the previous version i had

**Comment by amanzan** [comment]
Is the scrolling enabled in 2.1.1, or when will it be included?
